src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,11 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todos }) => {
```

Comment:
The `TodoList` component is missing type definitions for its `todos` prop. Consider defining a TypeScript interface or type for the `todos` array to ensure type safety and clarity.